specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/applications.json

```diff
@@ -0,0 +1,403 @@
+            "get": {
+                "tags": [
+                    "Applications"
+                ],
+                "description": "Lists properties of the specified application.",
+                "operationId": "Applications_Get",
+                "x-ms-examples": {
+                    "Get application on HDInsight cluster successfully created.": {
+                        "$ref": "./examples/GetApplicationCreated.json"
+                    },
+                    "Get application on HDInsight cluster creation in progress": {
+                        "$ref": "./examples/GetApplicationInProgress.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApplicationNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK. The request has succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/Application"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Applications"
+                ],
+                "description": "Creates applications for the HDInsight cluster.",
+                "operationId": "Applications_Create",
+                "x-ms-examples": {
+                    "Create Application": {
+                        "$ref": "./examples/CreateApplication.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApplicationNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ApplicationGetProperties"

```

Comment:
>ApplicationGetProperties [](start = 51, length = 24)

The name for the object is confusing. I think `ApplicationProperties` (without Get) should be a better option.

Reply:
Fixed.